### ISSUE TITLE
wont connect with go client

### ISSUE THREAD

**Issue by mathieujobin** [netvfy/go-netvfy-agent#29]
I can connect with snap client from ubuntu 22.04
but go client does not connect, neither Mac or Linux.

I have strace output pushed to keybase folder.

**Comment by nicboul** [comment]
yeah, the code is not ready yet, it's just a prototype being developed on OSX

**Comment by mathieujobin** [comment]
in 2021-2022, I thought I remembered it to work ...
